unlock-app/src/services/walletService.js

```diff
@@ -219,6 +219,17 @@ export default class WalletService extends EventEmitter {
     )
   }
 
+  async generateLockAddress() {
+    let transactionCount = await this.web3.eth.getTransactionCount(
```

Comment:
is this OK considering HardlyDifficult Nick's comments on the possible race condition? That's above my pay grade, just asking if that concern has been addressed

Reply:
Yeah, Julien and I discussed this. As this value is being used as a temporary address it is a non-issue at the moment.